tests/dataplicity/subcommands/test_version.py

```diff
@@ -0,0 +1,13 @@
+from dataplicity.subcommands.version import Version
+from dataplicity import __version__
+
+
+def test_version(capsys):
```

Comment:
this is the name of the dependency injection, as provided by http://doc.pytest.org/en/latest/capture.html plugin, not my invention :) I can assign a local variable inside the scope if you like:

`capture_sys = capsys`